test/NonSilo.Tests/AssemblyLoaderTests.cs

```diff
@@ -32,18 +34,6 @@ public void Dispose()
             this.defaultLoggerFactory.Dispose();
         }
 
-        [Fact, TestCategory("AssemblyLoader"), TestCategory("BVT"), TestCategory("Functional")]
-        public void AssemblyLoaderShouldDiscoverAssemblyLoaderTestAssembly()
```

Comment:
@ReubenBond is refactoring this tests to a better place. This test pass locally with updates

Reply:
Added https://github.com/dotnet/orleans/pull/4071